### ISSUE TITLE
VoiceOver does not read the role of "Start Debugging" button correctly and also name is missing for "Add Configuration"

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52332]
**Environment Details:**
VSCode Version : 1.24.0
OS Version : HighSierra10.13.5

**Additional Details:**
MAS Violated : MAS 1.3.1
ScreenReader : VoiceOver

**Repro Steps:**
1. Launch VS Code while VoiceOver is ON.
2. Navigate to Activity Bar and select "View Debug " button.
3. Navigate to debugging section toolbar under heading "Debug".

**Actual:**
Screenreader does not read the role of "Start Debugging" button correctly. It reads it as "group".
Name is missing for "Add Configuration" combobox beside that. Name is read as the value that is selected for the combobox.

**Expected:**
Screenreader should read the role correctly for "Start Debugging" button and name should be provided for "Add Configuration" combobox beside that.

**Recommendations:**
Provide proper name to the combobox and role to "Start Debugging" button.
or,
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
The screenreader users will not be able to know the correct name for the combobox to select the appropriate value and also will not be able to identify button present on the screen.

**MAS Reference:**
MAS 1.3.1 - https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachment For Reference:**
[A11y_VSCode_ViewDebug_VoiceOver_NameRoleIncorrect.pptx](https://github.com/Microsoft/vscode/files/2115394/A11y_VSCode_ViewDebug_VoiceOver_NameRoleIncorrect.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
I have added the role 'button' to the start debug.
However the name for the debug dropdown does not make much sense imho. We want to behave just like native dropdowns and for those reading out just the option makese the most sense to me. What else would you expect to be read here? @AccessibilityTestingTeam-TCS

**Comment by AccessibilityTestingTeam-TCS** [comment]
Yes options should definitely be read but along with what kind of options are available in the dropdown: should be added as name of the control. Screen reader users will come to know about the functionality/context of the control i.e. why is it exactly used for.
